### PR TITLE
Index

### PR DESCRIPTION
**Pull request by Micheledor**
Create /index view and add some basic front-end

### CODE REVIEW

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -1 +1,37 @@
 // Specific CSS for your home-page
+h2 {
+  text-align: center;  
+}
+
+#yacht-banner {
+  margin: auto;
+  margin-bottom: 20px;
+  position: static;  
+  background-image: url("https://images.unsplash.com/photo-1519011866958-6ac4dd2bbba4?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NTl8fHNlYSUyMHBhdHRlcm58ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60");
```
Comment (mifrugo) [inline app/assets/stylesheets/pages/_home.scss:10]: I would save it in the /app/assets/images folder.

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -2,5 +2,6 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [ :home ]
 
   def home
+    @yachts = Yacht.all
```
Comment (mifrugo) [inline app/controllers/pages_controller.rb:5]: Maybe in home we can show only the last five added?

`Yacht.limit(5).order('id desc')`

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -19,7 +19,7 @@
           <%= link_to "Messages", "#", class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
-          <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          <%= image_tag "https://thumbs.dreamstime.com/b/default-avatar-profile-icon-vector-social-media-user-image-182145777.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```
Comment (mifrugo) [inline app/views/shared/_navbar.html.erb:22]: As for the background, I would add a local image.
But In my merge I modified this part, we can leave it like this.

File: app/assets/stylesheets/pages/_home.scss (hunk 1 above)
Comment (Micheledor) [reply app/assets/stylesheets/pages/_home.scss:10]: Consider it done (tomorrow hehe)

File: app/controllers/pages_controller.rb (hunk 2 above)
Comment (Micheledor) [reply app/controllers/pages_controller.rb:5]: Yep, the logic I implemented was just to display some items in order to use them in the front-end. I guess we could do that 👌

**Pull request merged**